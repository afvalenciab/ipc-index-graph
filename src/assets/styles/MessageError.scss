.form__message-error {
  display: flex;
  align-items: center;
  margin-bottom: 16px;
  justify-content: center;
  background-color: rgba(219, 72, 72, 0.7);
  padding: 5px 0;
  border-radius: 5px;
  animation-name: scaleMessage;
  animation-duration: .3s;
  will-change: 'transform';

  figure {
    margin: 0;
    height: 30px;

    img {
      vertical-align: top;
      width: 30px;
      margin-right: 10px;
    }
    img path {
      fill: #fff;
    }
  }

  p {
    margin: 0;
  }
}

@keyframes scaleMessage {
  0%{
    transform: scale(0);
  }

  100%{
    transform: scale(1);
  }
}
