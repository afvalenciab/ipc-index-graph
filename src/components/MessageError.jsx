import React from 'react';
import '../assets/styles/MessageError.scss';
import imgError from '../assets/static/error.svg';

const MessageError = () => {
  return (
    <div className='form__message-error'>
      <figure>
        <img src={imgError} alt='Error Message Login' />
      </figure>
      <p>Usuario/Contrase√±a invalido! </p>
    </div>
  );
};

export default MessageError;
