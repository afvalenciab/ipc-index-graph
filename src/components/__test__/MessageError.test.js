import React from 'react';
import { create } from 'react-test-renderer';
import Adapter from 'enzyme-adapter-react-16';
import { render, configure } from 'enzyme';
import MessageError from '../MessageError';

configure({ adapter: new Adapter() });

describe('MessageError testing', () => {
  test('Match Snapshot', () => {
    const messageError = create(<MessageError />);
    expect(messageError.toJSON()).toMatchSnapshot();
  });

  test('MessageError has 1 img tag', () => {
    const messageError = render(<MessageError />);
    expect(messageError.find('img')).toHaveLength(1);
  });
});