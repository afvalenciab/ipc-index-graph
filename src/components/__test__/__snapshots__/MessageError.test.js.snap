// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MessageError testing Match Snapshot 1`] = `
<div
  className="form__message-error"
>
  <figure>
    <img
      alt="Error Message Login"
      src="test-file-stubs"
    />
  </figure>
  <p>
    Usuario/Contrase√±a invalido! 
  </p>
</div>
`;
